file/lib/file_source_endpoint.ex

```diff
@@ -110,36 +110,34 @@ defmodule Membrane.RTC.Engine.Endpoint.File do
 
   @impl true
   def handle_pad_added(Pad.ref(:output, {_track_id, _rid}) = pad, _ctx, state) do
-    payloader = Membrane.RTP.PayloadFormat.get(state.track.encoding).payloader
+    actions =
+      if state.track.encoding == :OPUS and
+           state.after_source_transformation == (&Function.identity/1) do
```

Comment:
One slight issue with this line:
```elixir
iex(1)> (&Function.identity/1) == (fn x -> x end)
false
```

But, provided that the users don't change the default identity function, I guess it should be fine